### ISSUE TITLE
[question] Possible mistake (or need for clarification) in documentation (section fetching sources from external repos)

### ISSUE THREAD

**Issue by SVincent** [conan-io/conan#10398]
I was following the documentation, when I was thrown off by a reference to a supposed explanation in the 'previous section' that doesn't seem to exist:

https://docs.conan.io/en/latest/creating_packages/external_repo.html

    "_There are two different ways to fetch the sources from an external repository:
    Using the source() method as we displayed in the previous section: [...]_"

I personally would read this sentence as "_There are two ways to fetch sources from an external repository, one of those is the source() method, we already displayed the source() method in the previous section_".
However no reference to a source() method seems to exist in the previous section, and no line containing `def source(self):` exists in conanfile.py.

For this reason I believe the sentence was maybe intended to be read as "_There are two ways to fetch sources from an external repository, one of those is the source() method. In the previous section we displayed the conanfile.py recipe, which we will now modify like this: [...]_"
An alternative explanation that I could think of is that source() really was displayed in the previous section of the documentation at some point, but this part got either removed or moved elsewhere, but the reference to the previous section remained.

Or did I accidentally skip over something or am misreading this sentence horribly?

- [x] I've read the [CONTRIBUTING guide](https://github.com/conan-io/conan/blob/develop/.github/CONTRIBUTING.md).

**Comment by memsharded** [comment]
You are right. The issue is that we updated the https://docs.conan.io/en/latest/creating_packages/getting_started.html page to start using the modern Conan integrations, but the next page of the docs is still not updated.

We will be doing a major effort in updating the docs now for Conan 2.0, we know the current state can be a bit confusing and inconsistent, but it is difficult to keep it in sync with all these changes in prep for 2.0, so sorry for that. Thanks for reporting, moving this to the docs repo.
